doc/examples/segmentation/plot_multiotsu.py

```diff
@@ -0,0 +1,76 @@
+"""
+=======================
+Multi-Otsu Thresholding
+=======================
+
+In this example we use multi-Otsu threshold to separate the pixels of an
+input image into three different classes, each one obtained according to
+the intensity of the gray levels within the image.
+
+Multi-Otsu calculates several thresholds, determined by the number of desired
+classes. The default number of classes is 3: for obtaining three classes, the
+algorithm returns two threshold values. They are represented by a red line in
+the histogram below.
+
+.. [1] Liao, P-S. and Chung, P-C., "A fast algorithm for multilevel
+thresholding", Journal of Information Science and Engineering 17 (5):
+713-727, 2001.
+"""
+
+from skimage.data import camera
```

Comment:
I prefer `from skimage import data` and below `data.camera()`. It's easier to test another image from our data set.